src/components/Navbar/Navbar.js

```diff
@@ -1,7 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Toolbar from './Toolbar';
+import SideNav from './SideNav';
 
 const Navbar = () => {
-  return <div>Navbar</div>;
+  const [navbarOpen, setNavbarOpen] = useState(false);
+
+  const handleToggleMenuClick = () => {
+    setNavbarOpen((prevState) => !prevState);
+  };
+  const handleToggleMenuNavClick = () => {
```

Comment:
this can be better names
```suggestion
  const handleCloseNavbar = () => {
```